go.mod

```diff
@@ -66,12 +66,12 @@ require (
 	github.com/mitchellh/go-wordwrap v1.0.1
 	github.com/mitchellh/hashstructure/v2 v2.0.2
 	github.com/np-guard/cluster-topology-analyzer v1.7.0
-	github.com/np-guard/netpol-analyzer v0.1.3
+	github.com/np-guard/netpol-analyzer v0.2.2
 	github.com/nxadm/tail v1.4.8
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b // indirect
-	github.com/openshift/api v3.9.1-0.20191201231411-9f834e337466+incompatible
+	github.com/openshift/api v0.0.0-20230502160752-c71432710382
```

Comment:
We need to make sure that CI is green after updating this. In the meantime, could you check whether we could maybe stick to the old version (3.9.1) for now?
I know it is very old, but the one `v0.0.0-20230...` looks like it would be a branch that is unreleased.

Reply:
Reverting to a 5-years old version will much complicate the netpol-analyzer code, we'd rather remove the openshift resources support if newer version cannot be supported. openshift/api do not make releases or tags in their github repo. 
All openshift repos that rely on openshift/api use recent pseudo versions in their go.mod (for example https://github.com/openshift/oc/blob/f26bd1970f012a0ee48df016d214e28cfa32cf2f/go.mod#L31 ).

Reply:
Okay, let's see first if this version works okay with the existing CI tests. If it does, then there is no need to revert that to 3.9.1